### ISSUE TITLE
Add support for @LongLink

### ISSUE THREAD

**Issue by piranna** [mafintosh/tar-stream#35]
GNU tar use [@LongLink](http://stackoverflow.com/questions/2078778/what-exactly-is-the-gnu-tar-longlink-trick) to set when the next entry will have a long name. This is used in the GCC and Linux kernel .tar.gz files, so when using tar-stream to de-compress them some files are not extracted and other times there are EACCESS errors.

**Comment by piranna** [comment]
```Javascript
/home/piranna/Proyectos/NodeOS/node_modules/nodeos-barebones/node_modules/nodeos-cross-toolchain/scripts/preinstall.js:35
  if(error) throw error;
                  ^
Error: EACCES, open '/home/piranna/Proyectos/NodeOS/node_modules/nodeos-barebones/node_modules/nodeos-cross-toolchain/deps/gcc/@LongLink'
    at Error (native)
```

**Comment by piranna** [comment]
I've found the same problem is happening [when extracting Node.js source code](https://github.com/kevva/download/issues/58#issuecomment-85975586):

```Javascript
#!/usr/bin/env node

var Download = require('download');
var progress = require('download-status');


// Source versions

const NODEJS_VERSION = "v0.11.14"


// Source URLs

const NODEJS_URL="http://nodejs.org/dist/"+NODEJS_VERSION+"/node-"+NODEJS_VERSION+".tar.gz"


Download({ extract: true, strip: 1 })
.get(NODEJS_URL, 'deps/node')
//.use(progress())
.run(function(error)
{
  if(error) throw error;

  console.log('Download finished!');
})
```

**Comment by mafintosh** [comment]
Fixed in 1.1.4 \o/

**Comment by piranna** [comment]
:+1:
